packages/components/src/ButtonDismiss/ButtonDismiss.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import styles from "./ButtonDismiss.css";
+import { Button } from "../Button";
+
+interface ButtonDismissProps {
+  onClick?(): void;
+  ariaLabel: string;
+}
+
+export function ButtonDismiss({ onClick, ariaLabel }: ButtonDismissProps) {
+  return (
+    <div className={styles.closeButtonWrapper}>
```

Comment:
I'm not a big fan of this wrapper technique if there's another way to avoid it.

But I wasn't able to assign more `classnames` directly to the `<Button>` component... must be an interface setup I've missed somewhere.

Reply:
We specifically **do not** want to assign `className` to `Button`. We want our components to be as opinionated as possible and allowing `className` lets people do some wild things.